eventemitter2.d.ts

```diff
@@ -11,12 +11,12 @@ declare module eventemitter2 {
          * @default '.'
          * @description the delimiter used to segment namespaces.
          */
-        delimiter?: string, 
```

Comment:
Yeah, but it seems that there's more lines affected which have changes to the whitespace.

I guess it doesn't really matter.